### ISSUE TITLE
test 3780: form for plans - accessibility

### ISSUE THREAD

**Issue by CarolingerSeilchenspringer** [liqd/a4-meinberlin#3395]
**URL:** dashboard, form for plans
**user:** initiator
**expected behaviour:** fully accessible
**behaviour:** not fully accessible
**important screensize:**
**device & browser:** 
**Comment/Question:** 

How far do we go for good accessibility? What can we implement and what is really complicated?
I checked to enter the form without mouse and this is what I can say:

firefox/ chrome: focus not well visible for Beschreibung, upload foto (both field and button), button "Abbrechen" (firefox only)
general: the map and address search is not really accessible either because the focus is hard to see or I don't know where i am at all.
safari: map not accessible at all, button "Abbrechen" without focus

**Comment by fuzzylogic2000** [comment]
Hmm, I think it is a good idea to improve it, but it doesn't seem to belong strictly to this story to me, because we didn't change any of it and the fields are used somewhere else as well. Is it necessary to be fixed in the next release?

**Comment by CarolingerSeilchenspringer** [comment]
@fuzzylogic2000 I don't know what the best way is to improve accessibility. It's one of the acceptance criteria as I thought that it is good to improve accessibility once we touch a form. But if these changes get too big or need platform wide changes, we have to find another solution. What do you think?

**Comment by fuzzylogic2000** [comment]
So, yes, I think for small things and for new fields, we can do it in the story and it is good to keep an eye out. 
But also yes, if it's bigger things, that is platform wide and also that was there before the story, it should either be tested before the story and defined in the story what exactly and how to improve it or solved differently. And this seems like a bigger thing to me.

**Comment by CarolingerSeilchenspringer** [comment]
ok, that sounds good. So we will close this issue and in new stories I will define more precisely how to improve accessibility.
